packages/vhd-lib/src/Vhd/VhdDirectory.js

```diff
@@ -100,16 +100,7 @@ export class VhdDirectory extends VhdAbstract {
     )
     // here we can implement compression and / or crypto
 
-    // chunks can be in sub directories :  create directories if necessary
-    const pathParts = partName.split('/')
-    let currentPath = this._path
-
-    // the last one is the file name
-    for (let i = 0; i < pathParts.length - 1; i++) {
-      currentPath += '/' + pathParts[i]
-      await this._handler.mkdir(currentPath)
-    }
-    return this._handler.writeFile(this._getChunkPath(partName), buffer, this._opts)
+    return this._handler.outputFile(this._getChunkPath(partName), buffer, this._opts)
```

Comment:
Does this really improve perf a lot?

Is this because `mkdir` calls, on S3, unnecessarily trigger a check for a file with the same name?

Reply:
yes, there was one trigger to check the existence of a directory per sub level ( so 2 additionnal HTTP request per upload) , and outputFile will  also create directories if they are needed .